File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    # oven = ''.join([str(num) for num in num_list if num%2 == 0])
    # odd = ''.join([str(num) for num in num_list if num%2 == 1])
    oven = ""
    odd = ""
    
    for num in num_list:
        if num%2 == 0:
            oven += str(num)
        else:
            odd += str(num)
    
    return int(oven) + int(odd) 

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        String sumOfOven = "";
        String sumOfOdd = "";
        
        for(int num:num_list) {
            if(num%2 == 0) {
                sumOfOven += String.valueOf(num);   
            }else {
                sumOfOdd += String.valueOf(num);
            }
        }
        // System.out.println(sumOfOven);
        // System.out.println(sumOfOdd);
        
        return Integer.valueOf(sumOfOven) + Integer.valueOf(sumOfOdd);
    }
}